drivers/storage/portworx/testspec/px-csi-win-storageclass.yaml

```diff
@@ -0,0 +1,12 @@
+allowVolumeExpansion: true
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: px-csi-win
```

Comment:
Should we name this to `px-csi-win-shared`?

Reply:
yes, that would make more sense, changes